### PR TITLE
Add subsciber model and POST endpoint

### PR DESCRIPTION
**Pull request by hexgnu**
# What

This resolves: https://www.pivotaltracker.com/story/show/97867620
https://www.pivotaltracker.com/story/show/97867630
https://www.pivotaltracker.com/story/show/97867614
# Testing

We added a few tests to stub out happy paths. Please comment if you think we need more test paths to visit.

### CODE REVIEW

File: spec/models/subscriber_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Subscriber, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (maxjustus) [inline spec/models/subscriber_spec.rb]: _COUGH_

**Comment by hexgnu** [comment]
@maxjustus juanna review this guy?

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# encoding: UTF-8
```
Comment (hexgnu) [inline db/schema.rb:1]: @maxjustus should we use SQL schema?

Comment (maxjustus) [inline db/schema.rb:1]: only if we use non-rialz-standard datatypes I stink

Comment (hexgnu) [inline db/schema.rb:1]: Yea I suppose for now it's not an issue so leave it as is.

**Comment by hexgnu** [comment]
Okay all updamafucated

File: app/api/made_to_measure/subscribers.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+module MadeToMeasure
+
```
Comment (maxjustus) [inline app/api/made_to_measure/subscribers.rb:2]: it's windy in here

Comment (maxjustus) [inline app/api/made_to_measure/subscribers.rb:2]: seems like this is a Chris style thing???

**Comment by maxjustus** [comment]
Such blend! Tested all the endpoints and I cried. My only feedback is on the stylistic thing with having newlines between class declarations and ends and stuff. Just makes me feel all spacey, but that should be a convo between Ewe Chriz and I.

File: app/api/made_to_measure/subscribers.rb (hunk 3 above)
Comment (hexgnu) [inline app/api/made_to_measure/subscribers.rb:2]: ja, he likes spaces. I kinda like no spaces in between things. 2 against 1 :wink:

**Pull request merged**